name:
    vicon: '/vicon/Jetson/pose'
    imu: 'imu/imu'
    xc: 'xc'
port:
    i2c: "/dev/i2c-0"
    address: [41,42,43,44]
node:
  Motor: 0
  MotorWarmup: 0
controller:
  flag_integral: false
  mode: 1
  g: 9.81
  m: 1.2 # 1.245 Jetson, 1.374 base odroid #1.7 with sensors
  J: [9.773E-3, 0.0, 0.0,
    0.0, 9.773E-3, 0.0,
    0.0,  0.0,  1.749E-2]
  del_t: 0.01
  R_conv: [1.0, 0.0, 0.0,
        0.0, -1.0, 0.0,
        0.0, 0.0, -1.0] # Convention conversion (NEU->NED)
  l: 0.225 # arm length odroid one 0.305
  c_tf: 0.1 # moment constant
  gain:
    att:
      kp: 0.67
      kd: 0.3
      ki: 0.0
      c: 0.10
    pos:
      kp: 16.0
      kd: 5.0
      ki: 2.0
      c: 0.10
  saturation:
    x: 3.0
    R: 1.0
